features/postmessage.yml

```
@@ -0,0 +1,16 @@
+name: PostMessage
+description: "The `postMessage()` method safely enables cross-origin communication for windows and workers, including popups and iframes."
+spec: https://html.spec.whatwg.org/multipage/web-messaging.html#web-messaging
+caniuse: x-doc-messaging
+group: messaging
+# Anchor baseline on window support.
+status:
+  compute_from: api.Window.postMessage
+compat_features:
+  # Already generated in Broadcast channel.
+  # Revisit once the "keys in impl or feature" question is answered.
+  #- api.BroadcastChannel.postMessage
+  - api.DedicatedWorkerGlobalScope.postMessage
+  - api.MessagePort.postMessage
+  - api.Window.postMessage
+  - api.Worker.postMessage
```

I think the `BroadcastChannel` and `MessagePort` equivalents are rather distinct, albeit confusingly named. `BroadcastChannel` is explicitly same origin. `MessagePort`'s `postMessage()` is explicitly non-global. I think I'd take this approach instead:

- Move `api.MessagePort.postMessage` to `channel-messaging.yml`
- Change the comment to say:

```suggestion
  # Other features, such as channel messaging and broadcast channels have
  # `postMessage` methods too, but are not covered by this feature.
  - api.DedicatedWorkerGlobalScope.postMessage
  - api.Window.postMessage
  - api.Worker.postMessage
```